ci/dhis2-verify-app.yml

```diff
@@ -64,6 +64,20 @@ jobs:
             - name: Lint
               run: yarn d2-style check
 
+    commitlint:
```

Comment:
This step is separate because it requires checking out all commits instead of the latest (which is the default).